tests/unittests/cmd/test_main.py

```diff
@@ -1,75 +1,80 @@
 # This file is part of cloud-init. See LICENSE file for license information.
 
 import copy
+import getpass
 import os
 from collections import namedtuple
-from io import StringIO
 from unittest import mock
 
 import pytest
 
 from cloudinit import safeyaml
 from cloudinit.cmd import main
 from cloudinit.util import ensure_dir, load_text_file, write_file
-from tests.unittests.helpers import FilesystemMockingTestCase, wrap_and_call
 
 MyArgs = namedtuple(
     "MyArgs", "debug files force local reporter subcommand skip_log_setup"
 )
 
 
-class TestMain(FilesystemMockingTestCase):
-    with_logs = True
-    allowed_subp = False
+class TestMain:
+    @pytest.fixture(autouse=True)
+    def common_mocks(self, mocker):
+        # cc_ubuntu_install module calls get_meta_doc on import.
+        # This call will fail if filesystem redirection mocks are in place
+        # and the module hasn't already been imported which can depend
+        # on test ordering.
+        mocker.patch("cloudinit.config.schema.get_meta_doc", return_value={})
```

Comment:
Now that https://github.com/canonical/cloud-init/pull/5757 landed I think this mock can be dropped right?
```suggestion
```